core/src/main/java/hudson/security/HudsonPrivateSecurityRealm.java

```diff
@@ -839,6 +839,10 @@ public Details newInstance(StaplerRequest2 req, JSONObject formData) throws Form
                     throw new FormException("Please confirm the password by typing it twice", "user.password2");
                 }
 
+                if (FIPS140.useCompliantAlgorithms() && pwd.length()< FIPS_PASSWORD_LENGTH) {
```

Comment:
Is the code formatted? For instance, there's no space between pwd.length() and <

Reply:
Formatting is not an issue here I guess.
We have no spotless issue , so it's fine in my opinion.

Reply:
that is not spotless issue doesn't mean we should not keep the same formatting as in the rest of the file. Please fix the white space
```suggestion
                if (FIPS140.useCompliantAlgorithms() && pwd.length() < FIPS_PASSWORD_LENGTH) {
```

Reply:
done